lhotse/dataset/signal_transforms.py

```diff
@@ -40,7 +40,7 @@ def from_file(cls, stats_file: Pathlike) -> "GlobalMVN":
     def to_file(self, stats_file: Pathlike):
         torch.save(self.state_dict(), stats_file)
 
-    def forward(self, features: torch.Tensor, *args, **kwargs) -> torch.Tensor:
+    def forward(self, features: torch.Tensor) -> torch.Tensor:
```

Comment:
can you add a `supervision_segments: Optional[torch.IntTensor] = None,` arg here to make it compatible with existing setups?

Reply:
adding